Formula/g/gpac.rb

```diff
@@ -31,27 +40,66 @@ class Gpac < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "3fa932d0f671a1643b20f381bba3181328431e5ef9555444a8d25a983ace36e1"
   end
 
+  depends_on "cmake" => :build
   depends_on "pkgconf" => :build
+  depends_on "a52dec"
+  depends_on "aom"
+  depends_on "curl"
+  depends_on "faad2"
+  depends_on "ffmpeg"
+  depends_on "freetype"
+  depends_on "jack"
+  depends_on "jpeg"
+  depends_on "libcaca"
+  depends_on "libnghttp2"
+  depends_on "libogg"
+  depends_on "libpng"
+  depends_on "libvorbis"
+  depends_on "lzip"
+  depends_on "mad"
+  depends_on "openjpeg"
   depends_on "openssl@3"
+  depends_on "pulseaudio"
+  depends_on "sdl2"
+  depends_on "theora"
+  depends_on "xvid"
 
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "alsa-lib"
+  end
+
+  resource "deps_unix" do
+    url "https://github.com/gpac/deps_unix.git", revision: "d2226b9ddf1715091b15b67dc8a799df65e8d576"
+  end
+
   def install
-    args = %W[
-      --disable-wx
-      --disable-pulseaudio
-      --prefix=#{prefix}
-      --mandir=#{man}
-      --disable-x11
-    ]
-
-    system "./configure", *args
-    system "make"
-    system "make", "install"
+    (buildpath/"gpac_public").install Dir["*"]
+    (buildpath/"deps_unix").install resource("deps_unix")
+
+    cd "deps_unix" do
+      ENV["CMAKE_POLICY_VERSION_MINIMUM"] = "3.5"
+      system "./build_libcaption_static.sh"
+    end
+
+    cd "gpac_public" do
+      args = %W[
+        --prefix=#{prefix}
+        --mandir=#{man}
+      ]
+
+      system "./configure", *args
+      system "make", "-j", ENV.make_jobs.to_s
```

Comment:
```suggestion
      system "make"
```
You don't need this.